src/plugins/kernel_install/systemd_bootloader.sh

```diff
@@ -0,0 +1,51 @@
+SYSTEMD_BOOT_ENTRIES_DIR="/boot/loader/entries"
```

Comment:
Add a simple doc on top of this file. It could be a good idea to point to some links in this doc.

Also, use single quotes around `/boot/loader/entrie`.